LandMark/Feature/Home/HomePage/HomePageViewModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  HomePageViewModel.swift
+//  LandMark
+//
+//  Created by Lucas Migge on 30/10/24.
+//
+
+import SwiftUI
+
+extension HomePageView {
+    class ViewModel: ObservableObject {
+        
+        let coordinator: HomeCoordinator
```

Comment:
Isso vai gerar uma referência cíclica, não?
![Image](https://github.com/user-attachments/assets/a59c1298-7979-4e6d-b0d9-24a59314a1c6)
```suggestion
        private(let) weak var coordinator: HomeCoordinator
```

Reply:
Vai sim. Acho que me distrai nesse ponto =/